documentation/configuration.md

```diff
@@ -727,6 +727,8 @@ In addition to enabling custom providers, `dotnet monitor` also allows you to di
 | queueAccountUri | string | false | The URI of the Azure queue account.|
 | metadata | Dictionary<string, string> | false | A mapping of metadata keys to environment variable names. The values of the environment variables will be added as metadata for egressed artifacts.|
 
+***Note:*** All built-in metadata keys are prefixed with `DotnetMonitor_`; to avoid metadata naming conflicts, avoid prefixing your metadata keys with `DotnetMonitor_`.
```

Comment:
```suggestion
***Note:*** Starting with `dotnet monitor` 7.0, all built-in metadata keys are prefixed with `DotnetMonitor_`; to avoid metadata naming conflicts, avoid prefixing your metadata keys with `DotnetMonitor_`.
```

Reply:
This is a breaking change, no matter how minor it might be, and cannot be backported to 6.x. I'll start a breaking changes document to capture the very few that we have.

Reply:
Good call - I forgot about that.